### DISCUSSION TITLE
whisper: The term 'whisper' is not recognized as the name of a cmdlet, function, script file, or operable program.

### DISCUSSION THREAD

**Discussion by Alex-coder251** [openai/whisper#1437] [Q&A] [answered]
i install it in win11, and this requirement have been satified: python3.10, ffmpeg and pytorch. 

i can find openai-whisper in `pip list`, but i don't know why i still can't use it.
![image](https://github.com/openai/whisper/assets/58896036/ecefe6dc-149e-416e-bdf1-08d3fbd60477)

are there someone who can help me?🥹

**Comment by m986883511** [comment]
一些建议
some advice
- 不要用中文问问题，不是国人项目，翻译一下，这是尊重
- do not use chinese ask queston, ，no chinese project, translate it ，it is respect
- 不要传很多图片，文字输出用就给文字输出，学一下markdown语法，github支持的
- do not up too many picture, you can give text output, learn markdown, github support it
- 不要在win11的裸环境上安装任何大的项目，你应该将它安装在虚拟环境中，学一下miniconda3
- do not install any big python project in win11 's python, you should install it in virtual env, i advice you use miniconda3

至于你的问题，明显是安装不正确，图片并不能代表安装成功，你可以打开一个powershell窗口，输入以下测试，只有命令行能用才代表安装ok
about your question, you install is not correct, picture can not prove install ok, open a powershell window, input it to test it. should use cli  to test it weather is installed.
```
pip list  |grep whisper
git --version
ffmpeg -version
python --version
```

我建议你修改你的提问为英文，并在之后的提文中使用英文
i advice you modify your quesiton to en, and use en to ask 

你看吧，我的英文，也很垃圾，但我问题基本都有答复，建议看一下stackoverflow关于提问的艺术。
简而言之就是提供更多的信息，将描述写清楚，注意排版，尽量能直接减少沟通的次数，你现在这问题问的在stackoverflow会被直接下架的

---无聊早上的回复

**Reply by Alex-coder251** [reply]
别骂了别骂了, 我不是看Discussions里面也有中国人就偷懒了嘛🙈

**Comment by m986883511** [comment]
> i install it in win11, and this requirement have been satified: python3.10, ffmpeg and pytorch.
> 
> i can find openai-whisper in `pip list`, but i don't know why i still can't use it. ![image](https://user-images.githubusercontent.com/58896036/244907284-ecefe6dc-149e-416e-bdf1-08d3fbd60477.png)
> 
> are there someone who can help me?🥹

this conde show it create a whisper.exe in your python Scripts dir
https://github.com/openai/whisper/blob/main/setup.py#LL38C1-L40C7

goto {python-env-path}\Scripts, check whisper.exe in it.

**Reply by Alex-coder251** [reply]
i have checked it out, but didn't find  whisper.exe
```

    目录: D:\WpSystem\S-1-5-21-3666208441-3826014804-3200322690-1001\AppData\Local\Packages\PythonSoftwareFoundation.Py
    thon.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\whisper


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
da----         2023/6/10     13:30                assets
da----         2023/6/10     13:30                normalizers
da----         2023/6/10     13:30                __pycache__
-a----         2023/6/10     13:30           4994 audio.py
-a----         2023/6/10     13:30          32702 decoding.py
-a----         2023/6/10     13:30          10970 model.py
-a----         2023/6/10     13:30          11783 timing.py
-a----         2023/6/10     13:30          12367 tokenizer.py
-a----         2023/6/10     13:30          22115 transcribe.py
-a----         2023/6/10     13:30           3583 triton_ops.py
-a----         2023/6/10     13:30           9686 utils.py
-a----         2023/6/10     13:30             26 version.py
-a----         2023/6/10     13:30           6844 __init__.py
-a----         2023/6/10     13:30             38 __main__.py
```

**Reply by phineas-pta** [reply, marked as answer]
don't look at `...\Python310\site-packages`

you should check `Python310\Scripts` or `Python310\bin`

**Reply by Alex-coder251** [reply]
thank you. i find it. than what should i do? add it to the system variable?
```
目录: D:\WpSystem\S-1-5-21-3666208441-3826014804-3200322690-1001\AppData\Local\Packages\PythonSoftwareFoundation.Py
    thon.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\Scripts


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----         2023/4/22     10:27         108475 black.exe
-a----         2023/4/22     10:27         108476 blackd.exe
-a----         2023/6/10     13:54         108469 coloredlogs.exe
-a----         2023/6/10     14:13         108507 convert-caffe2-to-onnx.exe
-a----         2023/6/10     14:13         108507 convert-onnx-to-caffe2.exe
-a----         2023/6/10     13:54         108484 ct2-fairseq-converter.exe
-a----         2023/6/10     13:54         108483 ct2-marian-converter.exe
-a----         2023/6/10     13:54         108488 ct2-openai-gpt2-converter.exe
-a----         2023/6/10     13:54         108487 ct2-opennmt-py-converter.exe
-a----         2023/6/10     13:54         108487 ct2-opennmt-tf-converter.exe
-a----         2023/6/10     13:54         108484 ct2-opus-mt-converter.exe
-a----         2023/6/10     13:54         108489 ct2-transformers-converter.exe
-a----         2023/6/10     12:46         108471 f2py.exe
-a----         2023/6/10     13:54         108494 huggingface-cli.exe
-a----         2023/6/10     13:54         108471 humanfriendly.exe
-a----         2023/4/22     10:28         108498 isort-identify-imports.exe
-a----         2023/4/22     10:28         108464 isort.exe
-a----         2023/6/10     12:46         108460 isympy.exe
-a----         2023/6/10     12:46         108499 normalizer.exe
-a----         2023/6/10     12:46            295 numba
-a----         2023/6/10     13:54         108488 onnxruntime_test.exe
-a----         2023/6/10     13:54         108465 pyav.exe
-a----         2023/6/10     14:13         108475 torchrun.exe
-a----         2023/6/10     12:45         108462 tqdm.exe
-a----         2023/6/10     13:30         108470 whisper.exe
```

**Reply by phineas-pta** [reply]
yeah you can

**Reply by Alex-coder251** [reply]
i did it. thank you for you help!🥳😍

**Comment by m986883511** [comment]
do not use wp python, so many problem...install miniconda3, create a venv, reinstall whisper in it
